Assets/_BForBoss/_Character/Scenes/Sandbox.unity

```diff
@@ -3103,5 +3103,9 @@ PrefabInstance:
       propertyPath: _camera
       value: 
       objectReference: {fileID: 1280265869}
+    - target: {fileID: 7177967516030943659, guid: 9cef2f6c7c3da6449a278b7a673ecbd2, type: 3}
```

Comment:
i can revert this if you want @unoctium1 , i created the custom input action for my dash.

Reply:
It's entierly up to you. so i'll update the prefab